README.md

```diff
@@ -38,14 +44,14 @@ This process refers to building the operator in a way that it can be installed l
    $ make bundle-image-push
    ```
 
-2. Build and push the image index for operator-registry:
+2. Build and push the image index for `operator-registry`:
    ```sh
    $ export INDEX_IMG=<registry>/<username>/external-dns-operator-bundle-index:1.0.0
    $ make index-image-build
    $ make index-image-push
    ```
 
-3. Create the catalogsource (registry secret may need to be linked to the external-dns-operator's pod created in `openshift-marketplace`):
+3. Create the `Catalogsource` object (you may need to link the registry secret to the pod of `external-dns-operator` created in `openshift-marketplace`):
```

Comment:
Is this - (you may need to link the registry secret to the pod of `external-dns-operator` created in `openshift-marketplace`)  optional for the developer? and is openshift-marketplace the namespace? If yes, where is it created as part of the procedure?